docs/concept-exercises.md

```diff
@@ -51,30 +51,51 @@ An exercise has the following files. In the browser they will show at the releva
+**Purpose:** Introduce the concept(s) that the exercise teaches to the student.
 
-As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes". Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.
+- The information provided should give the student just enough context to figure out the solution themselves. 
+- It should not give the student any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
+- Use the proper technical terms so that the student can easily search for more information. 
+- If the exercise introduces new syntax, an example of the syntax should always be included; students should not need to search the web for examples of syntax.
+
+As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes", tell the users how to create a string, and explain that a string has methods that can be used to manipulate it. Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.
 
 See the C# floating-point-numbers exercise's [introduction.md file][csharp-docs-introduction.md] for an example.
 
 ### `.docs/instructions.md`
 
-This file contains instructions for the exercise. It should explicitly explain what the student needs to do (define a method with the signature `X(...)` that takes an A and returns a Z), and provide at least one example usage of that function. If there are multiple tasks within the exercise, it should provide an example of each.
+**Purpose:** Provide instructions for the exercise.
+
+This file is split into two halves. 
+1. The first half explains the "story" or "theme" of the exericise. It should generally contain no code samples.
+2. The second half provides clear instructions of what a student needs to do, providing tasks.
+
+Each tasks must conform to the following standard:
+- Start with a third-level heading starting with a number (.e.g `### 1.`, `### 2.`)
+- Provide a method that a student needs to define with its signature(e.g. `X(...)` that takes an A and returns a Z),
```

Comment:
```suggestion
- Provide a method that a student needs to define with its signature (e.g. `X(...)` that takes an A and returns a Z),
```

Shouldn't this go into the exercise stub file?

This is track specific and I'm not sure if it should be considered in this file but on the Julia track we don't necessarily want to provide (type) signatures because it's often not a good practice to write them out. Would it be sufficient to provide something like "define method `foo` that takes two arguments" instead?

Reply:
Additionally, I could see this might get in the way of having useful exercises for some concepts, e.g. functions, which might require a student to parse such a requirement from prose and implement it

Reply:
I didn't immediately understand what this rule wanted me to do. Maybe rephrase to something like:

```suggestion
- Describe which function/method the student needs to define/implement (e.g. `Implement method X(...)` that takes an A and returns a Z),
```

Reply:
> Additionally, I could see this might get in the way of having useful exercises for some concepts, e.g. functions, which might require a student to parse such a requirement from prose and implement it. (i.e. writing the signature is the exercise)

@wolf99 Good point and I agree, but I'm going to leave this out of here, as it's a one-off exception to the rule, which I hope maintainers will be able to extrapolate.